csrc/transformer/inference/csrc/normalize.cu

```diff
@@ -99,7 +100,7 @@ __global__ void fused_bias_residual_layer_norm(__half* output,
     int gid = id >> 5;
     int warp_num = iteration_stride >> 5;
 
-    __half2 inp_reg[NORM_REG];
+    __half2 inp_reg[8];
```

Comment:
Replace `8` with symbolic constant?

Reply:
Thanks Tunji, I think this should be NORM_REG. It was not intentional, just for debugging. I will change it back